packages/Algernon.package/ALGExpressionType.class/instance/getDisplayedResult.st

```diff
@@ -0,0 +1,11 @@
+accessing
+getDisplayedResult
+
+	|command equationsResult|
+	
+	command := self getCommand asString.
+	
+	self isEquation
+		ifTrue: [equationsResult := Compiler evaluate: command for: self logged: false. 
```

Comment:
Maybe a try catch here to prevent unwanted error messages from showing up and making Algernon unusable.